src/app/components/core/developer/additional-services/additional-service-modal/additional-service-modal.component.ts

```diff
@@ -70,18 +75,31 @@ export class AdditionalServiceModalComponent {
     private dialogService: DialogService,
     private fileUploaderService: FileUploaderService,
     private configurationControlService: ConfigurationControlService,
-    public rolesService: RolesService) { }
+    private additionalServicesUtils: AdditionalServicesUtils,
+    public rolesService: RolesService) {
+      this.activatedRoute.queryParams.subscribe(params => {
+        if (params?.name) {
+            this.fromNavbar = true;
+            this.getServiceInfo(params, params?.pollingScheduleID);
+            // Wait to get html page loaded
+            setTimeout(() => {
+              this.toggleModal(true);
+            }, 500);
```

Comment:
We should carefully use the set timeout because the time given is not a guarantee that page will be available in that time.